releng/org.eclipse.emf.diffmerge.coevolution.target/org.eclipse.emf.diffmerge.coevolution.target.tpd

```diff
@@ -29,7 +36,22 @@ location "https://download.eclipse.org/releases/2021-06" {
 	org.eclipse.uml2.sdk.feature.group lazy
 }
 
-location "https://download.eclipse.org/sirius/updates/releases/7.0.4/2021-06" {
+location EMF-DiffMerge "https://download.eclipse.org/diffmerge/nightly/master/emf-diffmerge-site/" {
```

Comment:
I think this one shall be removed as it is defined in the pom.xml.
see https://github.com/eclipse/org.eclipse.emf.diffmerge.coevolution/blob/master/Jenkinsfile#L20